packages/ecs-agent/ecs-agent.spec

```diff
@@ -51,6 +50,7 @@ Requires: %{_cross_os}iptables
 %prep
 %autosetup -Sgit -n %{gorepo}-%{gover} -p1
 %cross_go_setup %{gorepo}-%{gover} %{goproject} %{goimport}
+cp %{S:9} "agent/version/version.go"
```

Comment:
This makes me uncomfortable but :man_shrugging: I get why you're doing it.

Can you add a comment here to be more explicit?

Reply:
Yeah, I'm not *happy* about the problem/solution either. I think this is an improvement over revving a patch with release data - we can revisit this if upstream later permits `ldflags` as distributed.

I'll add a comment to explain what's going on here.

Reply:
Added a brief comment [in the latest push](https://github.com/bottlerocket-os/bottlerocket/compare/f48003ffa7c9cdf330f29fa3c23ac6c9775feb8c..4b86f67797fab71a311246a1a2c33c2800c6c850).